project/Build.scala

```diff
@@ -0,0 +1,186 @@
+import sbt._
+import Keys._
+import scala.io.Source
+import scala.scalanative.sbtplugin._
+import ScalaNativePlugin.autoImport._
+import org.scalafmt.sbt.ScalaFmtPlugin.autoImport._
+
+object Build {
+  val assembleScalaLibrary =
+    taskKey[Unit](
+      "Clone scala standard library from the scala repo and then applies overrides.")
+
+  val toolScalaVersion = "2.10.6"
+
+  val libScalaVersion = "2.11.8"
+
+  val baseSettings = Seq(
+    organization := "org.scala-native",
+    version := nativeVersion,
+    scalafmtConfig := Some(file(".scalafmt"))
+  )
+
+  val toolSettings =
+    baseSettings ++
+      Seq(
+        scalaVersion := toolScalaVersion
+      )
+
+  val libSettings =
+    baseSettings ++
+      ScalaNativePlugin.projectSettings ++
+      Seq(
+        scalaVersion := libScalaVersion,
+        nativeEmitDependencyGraphPath := Some(file("out.dot"))
+      )
+
+  lazy val util: Project =
+    Project(
```

Comment:
Don't use the `Project()` constructor. It mostly disappears in sbt 1.0. Especially it doesn't have the `settings` parameter anymore.